Makefile

```diff
@@ -269,11 +271,14 @@ all-containerized:
 	docker rm $(BUILD_CONTAINER_NAME)
 	docker rmi $(BUILD_CONTAINER_IMAGE):$(VERSION)
 
-# Make both linux and windows binaries
+# Make all platform binaries
```

Comment:
please move the comment to after the target, on the same line (see the tools targets at L500+) so that it will be picked up as a docstring when I add the help target like in aquarius